backend/typescript/services/implementations/schedulingService.ts

```diff
@@ -127,11 +139,15 @@ class SchedulingService implements ISchedulingService {
         size: scheduling.size,
         is_pickup: scheduling.isPickup,
         pickup_location: scheduling.pickupLocation,
+        day_part: scheduling.dayPart,
         start_time: scheduling.startTime,
         end_time: scheduling.endTime,
         status: scheduling.status,
         volunteer_needed: scheduling.volunteerNeeded,
+        volunteer_time: scheduling.volunteerTime,
         frequency: scheduling.frequency,
+        recurring_donation_id: scheduling.recurringDonationId,
```

Comment:
just making sure, so if `scheduling.recurringDonationId` does not exist, it makes it null?

Reply:
I think it would be undefined, but I can double check. Should I set its default value to 0 since it's type number so that it'll function the same way as if it was null?

Reply:
I think it would be undefined as well which is fine

Reply:
![Screen Shot 2021-11-24 at 9 03 31 PM](https://user-images.githubusercontent.com/38363056/143365007-bce5ea41-b0de-458d-a6a7-347cd00c5ad9.png)
Looks like it's `null` in the db :))